pkgs/top-level/python-packages.nix

```diff
@@ -26943,4 +26944,41 @@ in modules // {
       license = licenses.wtfpl;
     };
   };
+
+  # tensorflow is built from a downloaded wheel, because the upstream
+  # project's build system is an arcane beast based on
+  # bazel. Untangling it and building the wheel from source is an open
+  # problem.
```

Comment:
I switched it to use `format = "wheel";`. Do you want me to also take the comment out? I'd like to leave it in, because there is unfinished work here, and it would be better to eventually build from source - for example, because then we wouldn't need the preFixup stage below.